config/routes.rb

```diff
@@ -6,8 +6,7 @@
     resources :contacts
   end
 
-  resources :characteristics, only: [:show]
-
+  resources :characteristics
```

Comment:
Maybe modify this to `resources :characteristics, only: [:show, :edit, :update, :delete]` to avoid creating a route for characteristics#index, which we will not use